plugins/modules/vsphere_copy.py

```diff
@@ -45,6 +45,11 @@
       - The timeout in seconds for the upload to the datastore.
     default: 10
     type: int
+  diskformat:
+    description:
```

Comment:
We should document when this has been added:

```suggestion
    version_added: 4.2.0
    description:
```

Well, hopefully 4.2.0.